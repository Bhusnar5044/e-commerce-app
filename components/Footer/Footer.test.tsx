import { render, screen } from '@test-utils';
import { Footer } from './Footer';

describe('Footer', () => {
    it('should render properly', () => {
        render(<Footer />);
        expect(screen.getByText(/2023 E-Commerce-Store, Inc. All rights reserved/i)).toBeInTheDocument();
    });
});
