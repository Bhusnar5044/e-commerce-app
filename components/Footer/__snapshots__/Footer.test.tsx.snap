// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render properly 1`] = `
<div>
  <footer
    class="bg-white border-t"
  >
    <div
      class="mx-auto py-10"
    >
      <p
        class="text-center text-xs text-black"
      >
        Â© 2023 E-Commerce-Store, Inc. All rights reserved.
      </p>
    </div>
  </footer>
</div>
`;
